packages/commerce-sdk-react/src/hooks/ShopperProducts/types.ts

```diff
@@ -0,0 +1,28 @@
+import {QueryParams} from '../../types'
+
+interface ShopperProductParams extends QueryParams {
+    id?: string
+    inventoryIds?: string
+    allImages?: boolean
+    perPricebook?: boolean
+}
+
+type ShopperProductsParams = Omit<ShopperProductParams, 'id'> & {
+    ids?: string[]
+}
+
+interface ShopperCategoryParams extends QueryParams {
+    id?: string
+    levels?: number
+}
+
+type ShopperCategoriesParams = Omit<ShopperProductParams, 'id'> & {
+    ids?: string[]
+}
+
+export type {
+    ShopperProductParams,
+    ShopperProductsParams,
+    ShopperCategoryParams,
+    ShopperCategoriesParams,
+}
```

Comment:
I think it's a very bad idea to define our own types for all these when we know they are defined, although not exported, in `commerce-sdk-isomorphic`. 

The approach we should take is to make a contribution to that library to export them. Another bad idea, but still better than defining our own similar types is to simply copy their types straight up.

Reply:
@bendvc i agree with using the types from `commerce-sdk-isomorphic`... There are many types we could use by just adding `export` to the `commerce-sdk-isomorphic`

However, there are also a lot of cases where the types are not defined but used inline. I'm not sure what's the best way to resolve those. let's chat.